themes/aframe/source/css/_settings.styl

```diff
@@ -37,3 +37,6 @@ $copy-link-hover = #35c5e9  // hsl(192, 80%, 56%)
 
 // Miscellaneous
 $radius = 2px
+
+// Margins
+$margin = 15px
```

Comment:
Could be more specific on the type of margin, and perhaps contain to the `home.styl` so we can clean up later.